src/modules/squeeze/squeeze.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable, UnprocessableEntityException } from '@nestjs/common';
+import { CreateSqueezeDto } from './dto/create-squeeze.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Squeeze } from './entities/squeeze.entity';
+
+@Injectable()
+export class SqueezeService {
+  constructor(
+    @InjectRepository(Squeeze)
+    private squeezeRepository: Repository<Squeeze>
+  ) {}
+
+  async createSqueeze(createSqueezeDto: CreateSqueezeDto) {
```

Comment:
this method was also implemented in pull request #400 , seems you guys have duplicate tickets